gnome/gobject-introspection/files/patch-fix-scanner-in-build-execution.diff

```diff
@@ -0,0 +1,61 @@
+--- giscanner/ccompiler.py.orig
++++ giscanner/ccompiler.py
+@@ -184,7 +184,10 @@
+         if os.name == 'nt':
+             runtime_path_envvar = ['LIB', 'PATH']
+         else:
+-            runtime_path_envvar = ['LD_LIBRARY_PATH', 'DYLD_FALLBACK_LIBRARY_PATH']
++            if sys.platform == "darwin":
++                runtime_path_envvar = ['DYLD_LIBRARY_PATH']
++            else:
++                runtime_path_envvar = ['LD_LIBRARY_PATH']
+             # Search the current directory first
+             # (This flag is not supported nor needed for Visual C++)
+             args.append('-L.')
+--- giscanner/dumper.py.orig
++++ giscanner/dumper.py
+@@ -236,8 +236,25 @@
+ 
+         args.extend(sources)
+ 
+-        pkg_config_libs = pkgconfig.libs(self._packages,
++        pkg_config_libs, pkg_config_libs_only_L = pkgconfig.libs(self._packages,
+                                          msvc_syntax=self._compiler.check_is_msvc())
++        this_L = []
++        for _pc_libs_L in pkg_config_libs_only_L:
++            this_L.append(_pc_libs_L[len('-L'):])
```

Comment:
```python
this_L = [lib[len('-L':)] for lib in pkg_config_libs_only_L]
```

does the same thing, is shorter, and arguably more idiomatic python.

Reply:
oh nice! I'm not a Python guru ... I know the basics & can sometimes come up with these shortcuts ... sometimes not. Thx!

Reply:
Minor fix: `len('-L':)` -> `len('-L'):`